### DISCUSSION TITLE
Does Rave Training stop by itself or should I do it by myself

### DISCUSSION THREAD

**Discussion by Federico8691** [acids-ircam/RAVE#130] [Q&A] [answered]
Dear friends, I am training a dataset made of percussions. I just reached 2.6 million points in 3 days of continuos processing. I do not remember if Rave stops by itself at a certain point, or if I should put an end to the training and to export the model. Any suggestion would be of great help. Thanks to everyone.

Federico Placidi

**Comment by AffeZwei** [comment, marked as answer]
If your size is set to default, train_rave.py will stop by itself at 3 Million steps.

**Reply by arpajik** [reply]
What are "steps" vs. Epoch? Where are the steps shown in the collab? Thanks for help.

**Reply by caillonantoine** [reply]
See #90

**Reply by caillonantoine** [reply]
regarding colab, you can monitor everything using tensorboard. A guide on how to interpret tensorboard metrics for a rave training is available here https://github.com/acids-ircam/RAVE/blob/master/docs/tensorboard_guide.md

**Reply by arpajik** [reply]
> If your size is set to default, train_rave.py will stop by itself at 3 Million steps.

Is it possible to safely stop and export the training before that 3 million steps?

**Reply by Federico8691** [reply]
Yes, you can just kill the process when you are satisfied with the output quality (you can listen to it over tensorboard.
